### PR TITLE
Zoom.ZoomOutlookPlugin version 5.15.5.926

### LINKED ISSUE #113739: [Update Request]: Zoom.ZoomOutlookPlugin
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

Zoom.ZoomOutlookPlugin

### Package Version

5.15.5.926

### Please describe the changes you would like to see

New version available 5.15.5.926

### PR DESCRIPTION
**Pull request by szhu25**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

Fix #113739
 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/113750&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/z/Zoom/ZoomOutlookPlugin/5.15.5.926/Zoom.ZoomOutlookPlugin.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Zoom.ZoomOutlookPlugin
+PackageVersion: 5.15.5.926
```
Comment (Trenly) [inline manifests/z/Zoom/ZoomOutlookPlugin/5.15.5.926/Zoom.ZoomOutlookPlugin.installer.yaml:5]:
```suggestion
PackageVersion: 5.15.5
```
This should match Control Panel

File: manifests/z/Zoom/ZoomOutlookPlugin/5.15.5.926/Zoom.ZoomOutlookPlugin.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Zoom.ZoomOutlookPlugin
+PackageVersion: 5.15.5.926
```
Comment (Trenly) [inline manifests/z/Zoom/ZoomOutlookPlugin/5.15.5.926/Zoom.ZoomOutlookPlugin.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 5.15.5
```
This should match Control Panel

File: manifests/z/Zoom/ZoomOutlookPlugin/5.15.5.926/Zoom.ZoomOutlookPlugin.locale.zh-CN.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.4.0.schema.json
+
+PackageIdentifier: Zoom.ZoomOutlookPlugin
+PackageVersion: 5.15.5.926
```
Comment (Trenly) [inline manifests/z/Zoom/ZoomOutlookPlugin/5.15.5.926/Zoom.ZoomOutlookPlugin.locale.zh-CN.yaml:5]:
```suggestion
PackageVersion: 5.15.5
```
This should match Control Panel

File: manifests/z/Zoom/ZoomOutlookPlugin/5.15.5.926/Zoom.ZoomOutlookPlugin.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Zoom.ZoomOutlookPlugin
+PackageVersion: 5.15.5.926
```
Comment (Trenly) [inline manifests/z/Zoom/ZoomOutlookPlugin/5.15.5.926/Zoom.ZoomOutlookPlugin.yaml:5]:
```suggestion
PackageVersion: 5.15.5
```
This should match Control Panel

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by szhu25** [comment]
Fixed. Thank you @Trenly !

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**